dali/kernels/signal/window/extract_windows_gpu.cu

```diff
@@ -21,11 +21,27 @@ namespace kernels {
 namespace signal {
 
 template <typename Dst, typename Src>
-KernelRequirements ExtractWindowsGpu<Dst, Src>::Setup(
+KernelRequirements ExtractWindowsGPU<Dst, Src>::Setup(
     KernelContext &context,
     const InListGPU<Src, 1> &input,
     const InTensorGPU<float, 1> &window,
```

Comment:
`window` is not used.

Reply:
That's the old way of doing things - but since we call Setup and Run by hand anyway, we might just as well start being rational and drop the unused parameters.